charts/istio-alerts/runbook.md

```diff
@@ -10,6 +10,23 @@ service. Many services have custom dashboards in DataDog as well which may help
 investigate this alert further, and most service also produce logs of requests
 which may provide more context into what errors are being returned and why.
 
+Can check trends/graph by:
```

Comment:
Should the runbook also provide action steps, such as: `if this, do that ; if this, ping that person ...`?

Reply:
yes! added in https://github.com/Nextdoor/k8s-charts/pull/324/commits/81512a45f8ac7346214459cec9f0462a542d182d